about.html

```diff
@@ -0,0 +1,167 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <!-- header starts-->
+    <header class="header">
+      <nav class="navbar">
+        <div class="top">
+          <ul class="container">
+            <li>
+              <a href="#"><i class="uil uil-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="uil uil-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#">English</a>
+            </li>
+            <li>
+              <a href="#">My Page</a>
+            </li>
+            <li>
+              <a href="#">Logout</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="bottom">
+          <div class="container">
+            <a href="index.html" class="nav-logo">KLWC</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campaign">CC Campaign</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- header ends -->
+
+    <!-- main starts -->
+
+    <main>
+      <section class="main-page about-page container">
+        <p class="greet about-hide">"Life-transforming events."</p>
+        <h1 class="about-header">brings together,<br/>
+          worshippers 2023</h1>
+        <hr />
+        <p class="description about-description">
+          "KLWC Free skill acquisation training.3 weeks free skills aquisation classes. Get empower!
+          seize the momment and get trained by renenowned facilitators for free!
+          
+        </p>
+        <a class="email" href="#">info@klwc_skills 2023.org</a>
+      </section>
```

Comment:
- [ ] As previous reviewer suggested, please ensure that your design looks the same as the given design. I appreciate your effort in doing so. In addition to what you have done, I would suggest that you align this contact address in the center as we can see on the design. Please look at the comparison 👇 
Given design
![image](https://user-images.githubusercontent.com/90222110/235215278-1969fc04-62d8-44a0-a056-6b63792827dc.png)

Your design
![image](https://user-images.githubusercontent.com/90222110/235215394-632e15b3-04f1-400f-ad54-c3a0a8909791.png)